README.md

```diff
@@ -112,6 +112,7 @@ See also `app/json-schema-validate.cpp`.
 
 ```C++
 #include <iostream>
+#include <iomanip>
```

Comment:
iomanip is needed? Where?

Reply:
setw is defined in iomanip https://en.cppreference.com/w/cpp/io/manip/setw
`+--> g++ -c -o main.o main.cpp -I ../json-schema-validator/src/ -I ../json/include/ -std=c++1z
main.cpp: In function ‘int main()’:
main.cpp:55:21: error: ‘setw’ is not a member of ‘std’
             << std::setw(2) << person << std::endl;`

Reply:
Oops; yes, you're right, I thought it was an addition you made to the `std::cerr` statement. Sorry.